cmd/sourced/compose/workdir/common.go

```diff
@@ -123,6 +123,10 @@ func ActivePath() (string, error) {
 		return "", err
 	}
 
+	if _, err := os.Lstat(path); err != nil {
+		return "", ErrMalformed.New("__active__", err)
```

Comment:
why is it `__active__` here and `active` 2 lines below?

Reply:
True,
also, it could be simplified by ae57867, to let the user know the underlying error (in order to differentiate if the link is broken, or missing)